lib/_kernel.scm

```diff
@@ -5212,8 +5217,8 @@ end-of-code
   ((c-lambda ()
              unsigned-int64
     "___return(___U64_add_U64_U64
-                 (___U64_mul_UM32_UM32 (___STAMP_YMD, 1000000),
-                  ___U64_from_UM32 (___STAMP_HMS)));")))
+                 (___U64_mul_UM32_UM32 (___STAMP_RELEASE_YMD, 1000000),
```

Comment:
The stamp `YMD` and `HMS` should be handled similarly to `VERSION`.  In other words `stamp.h` can override the stamp in `stamp-release.h`.  Some `#ifdef` logic is required at the top of `_kernel.scm`.

Reply:
Done.